adapters/smaato/smaato.go

```diff
@@ -84,7 +83,7 @@ func (adapter *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adap
 		return nil, []error{err}
 	}
 
-	isVideoEntryPoint := reqInfo.PbsEntryPoint == metrics.ReqTypeVideo
+	isVideoEntryPoint := reqInfo.PbsEntryPoint == "video" //never true
```

Comment:
Not sure what to do with this case. It will be never true because `eqInfo.PbsEntryPoint` cannot be `video`